doc/src/driver.rst

```diff
@@ -61,3 +61,76 @@ Verify each instruction
 .. option::  -r, --reduce
 
 Reduce program and verify
+
+roctx
+----
+
+.. program:: migraphx-driver roctx
+
+Provides marker information for each operation, allowing MIGraphX to be used with `rocprof <https://rocmdocs.amd.com/en/latest/ROCm_Tools/ROCm-Tools.html>`_ for performance analysis.
+This allows user to get GPU-level kernel timing information.
+An example command line combined with rocprof for tracing purposes is given below:
+
+::
+
+    /opt/rocm/bin/rocprof --hip-trace --roctx-trace --flush-rate 1ms --timestamp on -d <OUTPUT_PATH> --obj-tracking on /opt/rocm/bin/migraphx-driver roctx <ONNX_FILE> <MIGRAPHX_OPTIONS>
+
+After `rocprof` is run, the output directory will have many files which needs to be processed. In order to ease the processing output files,  **a helper script** is provided at `/tools/roctx.py`.
+The `roctx.py` helper script provides two main functionality: `run` and `parse`. Available knobs and usage are given below:
+
+::
+
+    Usage: roctx.py [-h] [--json-path json_path] [--out out]
+    [--study-name study-name] [--repeat repeat] [--parse]
+    [--run run] [--debug]
+
+.. option::  --run
+
+Runs MIGraphX with rocTX and given MIGraphX knobs, and parses the results, providing GPU kernel timing information.
```

Comment:
How does this run migraphx? What are MIGraphX knobs?

Reply:
It passes `roctx` command to `migraphx-driver`, and user can add additional migraphx-driver knobs, `--gpu`, `--batch 1` and so on.  I have changed the sentence here to reflect it is the `migraphx-driver` knobs.